server/game/cards/05_LOF/events/ImpossibleEscape.ts

```diff
@@ -0,0 +1,56 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+import { EventName, RelativePlayer, WildcardCardType } from '../../../core/Constants';
+
+export default class ImpossibleEscape extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '9069308523',
+            internalName: 'impossible-escape',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Exhaust a friendly unit or use the Force. If you do either, exhaust an enemy unit and draw a card',
+            optional: true,
```

Comment:
Do we know if `You may either` means that you can choose to do nothing? I would guess so but I'm not really sure.